### DISCUSSION TITLE
Which gem5 version is compatible with RVV 0.7

### DISCUSSION THREAD

**Discussion by mohammadvazirpanah** [gem5/gem5#694] [Q&A]
I want to use RVV 0.7, but I'm not sure which version of gem5 is compatible with this version. After checking and testing a few apps, it seems that the develop branch's final version can only support RVV 1.0.

**Comment by powerjg** [comment]
There is no upstream version of gem5 that supports RVV 0.7. There may be some forks out there somewhere, but they will not be supported by the gem5 maintainers.
